src/profile/Security.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <BasePage>
+    <template v-slot:header>
+      {{ $vuetify.lang.t('$vuetify.profile.index.2sv') }}
+    </template>
+
+    <ProfileProgress :profile="{user: $user, alternates, mfa}"/>
+
+
+    <v-subheader>
+      You’ll be asked for your second method every 30 days. Drag and drop to reorder method preference
```

Comment:
Do we want to leave out the "Drag and drop..." bit until we implement that feature (if we do so)?

Reply:
Yup.